nifi-nar-bundles/nifi-grpc-bundle/nifi-grpc-processors/pom.xml

```diff
@@ -75,6 +75,13 @@ language governing permissions and limitations under the License. -->
             <artifactId>nifi-mock</artifactId>
             <version>1.14.0-SNAPSHOT</version>
             <scope>test</scope>
+<!--The following is added to resolve dependencies issues bringing in AWS KMS dependency-->
```

Comment:
Recommend adjusting this comment to indicate the direct and transitive dependency more clearly:
```suggestion
            <!-- Exclude transitive dependency of software.amazon.awssdk:kms included in nifi-sensitive-property-provider -->
```

Reply:
Adjusting the comment